lib/modules/json/model/src/main/java/io/atlasmap/json/v2/JsonInspectionRequest.java

```diff
@@ -40,150 +42,162 @@
 
     protected InspectionType type;
 
+    protected List<String> inspectPaths;
```

Comment:
We can add `BaseInspectionRequest` abstract class (and maybe remove InspectionRequest interface if not needed) and add getter/setter for this there if it's data format agnostic.

Reply:
Fixed!